starlette/responses.py

```diff
@@ -404,7 +404,7 @@ async def _handle_multiple_ranges(
         file_size: int,
         send_header_only: bool,
     ) -> None:
-        boundary = "".join(random_choices("abcdefghijklmnopqrstuvwxyz0123456789", k=13))
+        boundary = token_hex(20)
```

Comment:
Why was it increased to 20?

Reply:
just random number I picked

Reply:
According to the specification, this value can be between 1 and 70. But is 40 too long?

Reply:
>96 bits of entropy is recommended

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type#directives

Reply:
Can we drop a bit the number then?

Reply:
@trim21 Can we drop the number a bit so I can include this on the next release?

Reply:
Yep. Maybe we can add a comment about the choice, to avoid future confusion/questions?

Reply:
> >96 bits of entropy is recommended so only 13 bytes is needed

where does this recommendation comes from?

Reply:
https://github.com/whatwg/html/issues/6424#issuecomment-826036018